src/Container/PHPDiFactory.php

```diff
@@ -18,9 +18,13 @@ class PHPDiFactory
     public static function buildContainer($config = [])
     {
         $builder = new DI\ContainerBuilder();
-        $builder->useAnnotations(true);
+        $builder->useAnnotations(false);
```

Comment:
This PR is WIP :D I don't know what is the future.. I'm trying to resolve the bootstrap of ->run function